### DISCUSSION TITLE
:tea: Would you consider deliver duckdb on  tea.xyz pantry :grey_question:

### DISCUSSION THREAD

**Discussion by adriens** [duckdb/duckdb#6721] [Ideas]
duckdb is a great tool, delivering it through [tea.xyz](https://tea.xyz/) would help distribute it even easier... even in [GH Acions](https://github.com/marketplace/actions/tea-setup) (we have started to use duckdb on our CI)

![image](https://user-images.githubusercontent.com/5235127/225177829-4dffbec6-3f85-4464-b6af-e8c556f393c7.png)

**Comment by Mause** [comment]
If this is something you want, you can submit the pr to the "pantry" yourself? It doesn't look like this is something we need to do for you

**Reply by adriens** [reply]
> If this is something you want, you can submit the pr to the "pantry" yourself?

Yes I can.

>  It doesn't look like this is something we need to do for you

Indeed, indeed

**Comment by adriens** [comment]
On its way : 

- https://github.com/teaxyz/pantry.extra/issues/780

**Comment by adriens** [comment]
![image](https://user-images.githubusercontent.com/5235127/225496226-dab571c7-10b3-448f-9535-e65414f162be.png)
https://tea.xyz/+duckdb.org/

**Comment by adriens** [comment]
- https://github.com/duckdb/duckdb-web/pull/642
